api/dbConfig/dev_seeds.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO posts (title, pname , body)
+VALUES 
+(' Futureproof', 'Sarushan', 'We are so happy at futureproof'),
+
+('Cablegram', 'Evie', 'Where is my damn duck?');
+
```

Comment:
So this file could just be added onto your posts.sql file and it would add when the docker is being composed.